lib/src/base-styles/_nx-btn.scss

```diff
@@ -7,111 +7,122 @@
 @import '../scss-shared/nx-variables';
 @import '../scss-shared/nx-text-helpers';
 
+%nx-btn--disabled {
+  &, &:hover, &:focus {
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
+    cursor: default;
+  }
+}
+
 .nx-btn {
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
   line-height: 16px;
-  margin: 0 8px;
-  padding: 0 12px;
+  margin: 0 $nx-spacing-xs;
+  padding: $nx-spacing-sm $nx-spacing-l;
   text-align: center;
   vertical-align: middle;
   min-width: 100px;
 
-  + .nx-btn {
-    // buttons should be 8px from adjacent items, except other buttons which they should only be 4px from
-    margin-left: -4px;
+  &:hover {
+    background-color: #c5deff;
   }
 
-  &:hover, &:focus, &:active {
-    border-color: $nx-slate;
-    color: $nx-text-color;
+  &:focus {
+    border: 1px solid #43cbf5;
+    outline: 0;
   }
 
   &:active {
-    background-color: $nx-grey-2;
+    background-color: #e8f2ff;
   }
 
   &.disabled, &[disabled] {
```

Comment:
See https://github.com/sonatype/sonatype-react-shared-components/commit/68ca7ea49ada98929d132a41cfbf49c531ba17a3 for a slight improvement to the pattern you have here

Reply:
> See [68ca7ea](https://github.com/sonatype/sonatype-react-shared-components/commit/68ca7ea49ada98929d132a41cfbf49c531ba17a3) for a slight improvement to the pattern you have here

Nice, thanks.